docs/quick-start.md

```diff
@@ -73,7 +73,6 @@ To setup Docker:
 - Open `tools/docker/.env` and make any modifications you'd like.	
 	- It's strongly recommend you at least change `WP_ADMIN_PASSWORD` to something more secure.
 - Start the Docker container using `jetpack docker up -d` (this may take some time for the first setup)
-- Install WordPress in your Docker container using `jetpack docker install` 
```

Comment:
Technically, you already got a fresh non-installed WP when starting Docker containers, but `jetpack docker install` [proceeds to install it, set up users, useful plugins and configurations](https://github.com/Automattic/jetpack/blob/ab5cd1b1ba99182c836470c3901f08a6e7d482e4/tools/docker/bin/install.sh). So I'd recommend keeping it.

```
╰─ jetpack docker install
Success: WordPress installed successfully.
Success: Updated 'blog_public' option.
Creating user: author
Creating user: editor
Creating user: subscriber
Creating user: contributor
Installing Query Monitor – The developer tools panel for WordPress (3.16.4)
Downloading installation package from https://downloads.wordpress.org/plugin/query-monitor.3.16.4.zip...
Unpacking the package...
Installing the plugin...
Plugin installed successfully.
Activating 'query-monitor'...
Plugin 'query-monitor' activated.
Success: Installed 1 of 1 plugins.
Installing WP Crontrol (1.17.0)
Downloading installation package from https://downloads.wordpress.org/plugin/wp-crontrol.1.17.0.zip...
Unpacking the package...
Installing the plugin...
Plugin installed successfully.
Activating 'wp-crontrol'...
Plugin 'wp-crontrol' activated.
Success: Installed 1 of 1 plugins.
Installing Gutenberg (19.0.0)
Downloading installation package from https://downloads.wordpress.org/plugin/gutenberg.19.0.0.zip...
Unpacking the package...
Installing the plugin...
Plugin installed successfully.
Activating 'gutenberg'...
Plugin 'gutenberg' activated.
Success: Installed 1 of 1 plugins.
Installing User Switching (1.8.0)
Downloading installation package from https://downloads.wordpress.org/plugin/user-switching.1.8.0.zip...
Unpacking the package...
Installing the plugin...
Plugin installed successfully.
Activating 'user-switching'...
Plugin 'user-switching' activated.
Success: Installed 1 of 1 plugins.
Success: Enabled 4 of 4 plugin auto-updates.
Plugin 'jetpack' activated.
Success: Activated 1 of 1 plugins.

WordPress installed. Open localhost
```

Reply:
Hi. Do you get this message after running `jetpack docker up -d`? All I get is:

```
$ jetpack docker install

WordPress has already been installed. Uninstall it first by running:

  jetpack docker uninstall

Command exited with status 1
```

Hence why the suggestion look redundant to me. Maybe we are using different flows? 🤔